downstream/modules/platform/proc-upgrade-installer.adoc

```diff
@@ -64,6 +72,21 @@ node_type=execution
 <1> Specifies a control node that runs project and inventory updates and system jobs, but not regular jobs. Execution capabilities are disabled on these nodes.
 <2> Specifies peer relationships for node-to-node connections in the `[execution_nodes]` group.
 <3> Specifies hop nodes that route traffic to other execution nodes. Hop nodes cannot execute automation.
+. Import or generate a {HubName} API token.
++
+* Import an existing API token with the `automationhub_api_token` flag:
++
+[source,options="nowrap",subs=attributes+]
+----
+automationhub_api_token=<api_token>
+----
++
+* Generate a new API token, and invalidate any existing tokens, with the `generate_automationhub_token` flag:
++
+[source,options="nowrap",subs=attributes+]
+----
+generate_automationhub_token=
```

Comment:
```suggestion
generate_automationhub_token=true
```
From "- set 'generate_automationhub_token' to True to generate a new token" in the ticket. Don't know whether it should be `true`, `"true"` or `True`.